### DISCUSSION TITLE
unable to tile a window that has been moved while in floating layout

### DISCUSSION THREAD

**Discussion by ekansh-agastya** [qtile/qtile#3485] [Q&A]
whenever i open a terminal or any other application in floating mode and then change if i change its position via mouse. the next time i change the layout to a tiling layout the window does not get tiled. it just appears on top of all other windows.

**Comment by elParaguayo** [comment]
Once you change the position with the mouse, the window becomes floating and is not managed by any tiling layouts. 

To get it back to being managed by the layouts, you need to bind a key to `lazy.window.toggle_floating()`.

**Reply by ekansh-agastya** [reply]
thanks a lot for replying back so quickly, and yeah it solved my problem
